src/gluonts/mx/model/deepvar_hierarchical/_estimator.py

```diff
@@ -73,7 +73,10 @@ def constraint_mat(S: np.ndarray) -> np.ndarray:
     return A
 
 
-def null_space_projection_mat(A: np.ndarray) -> np.ndarray:
+def null_space_projection_mat(
+    A: np.ndarray,
+    D: Optional[np.ndarray],
```

Comment:
Why do we give this matrix as `optional`?
What are the advantages against `D: np.array = None`? With the current approach tests are failing.

Reply:
ya, `=None` is missing actually. Let me fix.